app/[lang]/layout.tsx

```diff
@@ -6,12 +6,12 @@ import { notFound } from 'next/navigation';
 import { hasLocale, Locale, NextIntlClientProvider } from 'next-intl';
 import { ReactNode } from 'react';
 
-import Footer from '~/components/Footer/Footer';
 import Header from '~/components/Header/Header';
 import { theme } from '~/ds-components/theme/Theme';
 import ThemeProvider from '~/ds-components/theme/ThemeProvider';
 
 import { routing } from '~/i18n/routing';
+import Footer from '~/shared/components/Footer/Footer';
```

Comment:
```suggestion
import Footer from '~/components/Footer/Footer';
```

We have aliases now, don`t forget to use them

Reply:
Thank you! Updated those